CHANGES.rst

```diff
@@ -149,6 +149,7 @@ rscd
 ----
 
 - Added baseline algorithm that flags groups [#4669]
+- Fixed bug when the READPATT/SUBARRAY data is not found in ref file [#4934]
```

Comment:
This needs to go up at the top of the file, under the 0.16.1 release (not down here in 0.16.0, which is already gone)